src/main/java/ch/ejpd/lgs/searchindex/client/service/sync/AbstractSyncService.java

```diff
@@ -63,6 +148,24 @@ public int processFullQueuePaging(
           return 0;
         }
 
+        boolean isAnyUsingLandRegister =
```

Comment:
unclear variable name. isUsingLandRegisters?
Also, I don't understand Map<String, Integer> landRegisterCounts. What's the count? Pages per landRegister?

Reply:
person data per land register